pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.props

```diff
@@ -42,7 +42,6 @@
       <Platform>armel</Platform>
     </OfficialBuildRID>
     <OfficialBuildRID Include="debian.8-x64" />
-    <OfficialBuildRID Include="fedora.23-x64" />
     <OfficialBuildRID Include="fedora.24-x64" />
```

Comment:
@ellismg would know more, but I thought this defined the officially supported platforms. If Fedora 25 is not supported, it shouldn't be here. I seem to recall reading/hearing that Fedora 25 will be supported via the portable linux build.